src/docs/product/profiling/getting-started.mdx

```diff
@@ -9,7 +9,7 @@ description: 'Get started with Profiling, which allows you to see code-level pro
 
 <Note>
 
-Profiling is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [profiling@sentry.io](mailto:profiling@sentry.io).
+If you have any questions or feedback, please email us at [profiling@sentry.io](mailto:profiling@sentry.io).
```

Comment:
I would remove this block altogether. We want our users to reach out via GitHub, creating issues in the respective SDKs. This will also allow people to search for similar problems.